applications/dashboard/modules/class.pagermodule.php

```diff
@@ -64,14 +64,14 @@ class PagerModule extends Gdn_Module {
     public $Offset;
 
     /** @var int The last offset of the current page. (ie. Offset to LastOffset of TotalRecords). */
-    private $_LastOffset;
+    protected $_LastOffset;
```

Comment:
Why going from private to protected? We want attributes to be private by default.